dtfs-central-api/src/v1/controllers/bank/get-banks.controller.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../../../drivers/db-client');
+
+exports.getAllBanksGet = async (req, res) => {
```

Comment:
I can see your following convention here and probably worth keeping this in but the double get just seems unnecessary and silly

Reply:
Yeah, I had the same feeling while writing it, but went with this in the end to follow the convention.